bridge/contracts/DepositNotifier.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT-open-group
```

Comment:
Can we have this a separate PR?

Reply:
Seems like after all the many merges and rebases of this PR, I missed a thing or two, thanks!